tensorflow/core/kernels/BUILD

```
@@ -2413,6 +2412,7 @@ cc_library(
     hdrs = ["range_sampler.h"],
     visibility = ["//visibility:private"],
     deps = [
+        "//tensorflow/core:framework",
```

We can also remove the dependency here since `random_distributions.h` doesn't depend on it anymore.

---

Please also change this to `numeric_types`.
```suggestion
        "//tensorflow/core/framework:numeric_types",
```